src/components/CartPage.vue

```diff
@@ -51,21 +42,35 @@
       checkoutStatus () {
         return this.$store.state.cart.lastCheckout
       },
-      total () {
-        let totalDollar = 0
-//        Object.keys(this.carts).forEach(key => {
-//          for (var i = 0; i < this.carts[key].products.length; i++) {
-//            totalDollar += (this.carts[key].products[i].quantity * this.carts[key].products[i].price_cents / 100)
-//          }
-//        })
-        return totalDollar
+      hasNonEmptyCart () {
+        var check = false
+        Object.keys(this.carts).forEach(key => {
+          for (var i = 0; i < this.carts[key].products.length; i++) {
+            if (this.carts[key].products[i].quantity) {
+              check = true
+            }
+          }
+        })
+        return check
       }
     },
     methods: {
       ...mapActions([
         'checkout',
         'addToCart'
       ]),
+      formattedPrice (itemTotal) {
```

Comment:
little helper methods to avoid rewriting shit and updating 30 spots at the same time